tracer/src/Datadog.AutoInstrumentation.NativeLoader/instrumented_assembly_generator/instrumented_assembly_generator_cor_profiler_function_control.cpp

```diff
@@ -0,0 +1,89 @@
+#include "instrumented_assembly_generator_cor_profiler_function_control.h"
+#include "instrumented_assembly_generator_helper.h"
+#include "instrumented_assembly_generator_metadata_interfaces.h"
+#include "../log.h"
+
+namespace instrumented_assembly_generator
+{
+InstrumentedAssemblyGeneratorCorProfilerFunctionControl::InstrumentedAssemblyGeneratorCorProfilerFunctionControl(
```

Comment:
overall comment:
you do no need to add `InstrumentatedAssemblyGenerator` in all names those classes are defined in `instrumented_assembly_generator` (it's redundant).
Shorter (nonetheless expressive) names improve readability.

Reply:
Fixed in ebde2f57c8ef03fd167862f981abd5c4958c222a